dashboards/confluent-cloud/confluent-cloud.json

```diff
@@ -7,130 +7,128 @@
       "description": null,
       "widgets": [
         {
-          "title": "Number of Brokers",
+          "title": "",
```

Comment:
Looks like we don't have a title here. Is that intentional?

Reply:
@brammerl  Hmm, should there be a title for this one? It would basically be the same as the dashboard because it's a heading widget. I can add it.